lib/elixir_search_extractor_worker/keyword_search_worker.ex

```diff
@@ -0,0 +1,19 @@
+defmodule ElixirSearchExtractor.ElixirSearchExtractorWorker.KeywordSearchWorker do
+  use Oban.Worker
+  alias ElixirSearchExtractor.Repo
+  alias ElixirSearchExtractor.SearchKeyword.GoogleSearcher
+  alias ElixirSearchExtractor.SearchKeyword.Schemas.Keyword
+  alias ElixirSearchExtractor.SearchKeyword.SearchKeywords
+
+  @impl Oban.Worker
+  def perform(%Oban.Job{args: %{"keyword_id" => keyword_id}}) do
+    keyword_record = Repo.get_by!(Keyword, %{id: keyword_id})
+
+    with {:ok, attributes} <- GoogleSearcher.search(keyword_record),
```

Comment:
I'm not sure here as it might not clear the this is parsing the HTML as well 🤔  Maybe we can split them 💭